src/schema/mutation/editApiConfiguration.mutation.ts

```diff
@@ -79,7 +79,33 @@ export default {
       )
         .where({ _id: args.id })
         .getFilter();
-      const apiConfiguration = await ApiConfiguration.findOneAndUpdate(
+      let apiConfiguration;
+      apiConfiguration = await ApiConfiguration.findOne(filters);
+      // If there is a currently existing API configuration, get the previous value
+      // so we don't lost any other configuration that is not edited
+      if (apiConfiguration) {
+        const decrypt = await CryptoJS.AES.decrypt(
```

Comment:
@unai-reliefapp 
( no change required, just for info )
not working for new api configuration